docs/site/Discovering-models.md

```diff
@@ -8,6 +8,8 @@ permalink: /doc/en/lb4/Discovering-models.html
 
 ## Synopsis
 
+Please note that the code below will not work without creating .js connector files. Also not that lb4 will only create .ts connector files. Your on your own, good luck.
+
```

Comment:
Can you elaborate the issue you ran into? Did you refer to an existing issue?

Reply:
+1 to what Raymond said.

The information in the note does not match my mental model of how model discovery works, it does not make sense to me.

Let's build a better understanding of the problem you @SouthRibbleTech are facing. Then we can provide a better advice than just "you are on your own" and perhaps even improve our tooling to support your scenario.

Reply:
I'm making an assumption, but I believe the culprit is this line:

![image](https://user-images.githubusercontent.com/25147899/69114705-1b3a6880-0ac1-11ea-9007-a8cc717b6fc1.png)

While it does say the project must be built, it assumes that the readers know that they should run `npm run build`. Currently, there's no documentation on the default, generated npm scripts. The only documentation on them is on the [`@loopback/build` README ](https://github.com/strongloop/loopback-next/blob/master/packages/build/README.md).

We could potentially create a new doc page explaining the purpose of these scripts and how to use them, then link to it from the `Discovering Models` doc page.